.github/workflows/build.yml

```diff
@@ -29,12 +29,12 @@ jobs:
       matrix:
         targets: [esp32, esp32s2, esp32c3]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Build MCUboot bootloader
         uses: docker://docker.io/espressif/idf:release-v4.4
         with:
-          args: ./build_mcuboot.sh -s -c ${{matrix.targets}}
-      - uses: actions/upload-artifact@v2
+          args: /bin/sh -c "git config --global --add safe.directory '*' && ./build_mcuboot.sh -s -c ${{matrix.targets}}"
```

Comment:
When will we need to add this `safe.directory`?

Reply:
`git` now prevents accessing a directory which is not owned by the same user as the one from repository tree.
This happens only on MCUboot build because the IDF Docker container updates the `mcuboot` repository, which is only mounted on the container, but does not belong in it.